iris_grib/tests/unit/save_rules/__init__.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2013 - 2015, Met Office
+# (C) British Crown Copyright 2013 - 2016, Met Office
```

Comment:
Are these dates correct? Sure, we've had the code since 2013, but it's been Iris code for most of that...